scripts/serveDeployments.ts

```diff
@@ -0,0 +1,20 @@
+import express, { Express } from 'express';
+
+// Script to serve the local deployment addresses
+
+// For localhost testing:
+//    - Spin up a node (http://127.0.0.1:8545/ by default):
+//        `npx hardhat node`
+//    - Run the deploy script or create valid deployments.local.json file
+//        `npx hardhat run --network localhost scripts/deploy.ts`
+//    - Run this script:
+//        `npx ts-node scripts/serveDeployments.ts`
+
+const port = process.env.SERVE_PORT || 8080;
```

Comment:
The port and comment use different port numbers. Is that intended here?

Reply:
The comment is the node port, but the "serve" port is the port where a small server will be deployed to fetch the deployment.*.json files. For example, the integration tests use this endpoint to know where contract locations are